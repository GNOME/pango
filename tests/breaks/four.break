# For Thai language.
ภาษาไทย หรือ ภาษาไทยกลาง เป็นภาษาราชการและภาษาประจำชาติของประเทศไทย ภาษาไทยเป็นภาษาในกลุ่มภาษาไทซึ่งเป็นกลุ่มย่อยของตระกูลภาษาขร้า-ไท สันนิษฐานว่า ภาษาในตระกูลนี้มีถิ่นกำเนิดจากทางตอนใต้ของประเทศจีน และนักภาษาศาสตร์บางส่วนเสนอว่า ภาษาไทยน่าจะมีความเชื่อมโยงกับตระกูลภาษาออสโตร-เอเชียติก ตระกูลภาษาออสโตรนีเซียน และตระกูลภาษาจีน-ทิเบต
