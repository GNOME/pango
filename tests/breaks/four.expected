Text:         ภ า ษ า   ไ ท ย  [ ]   ห รื อ  [ ]   ภ า ษ า   ไ ท ย   ก ล า ง  [ ]   เ ป็ น   ภ า ษ า   ร า ช ก า ร   แ ล ะ   ภ า ษ า   ป ร ะ จ ำ   ช า ติ   ข อ ง   ป ร ะ เ ท ศ   ไ ท ย  [ ]   ภ า ษ า   ไ ท ย   เ ป็ น   ภ า ษ า   ใ น   ก ลุ่ ม   ภ า ษ า   ไ ท   ซึ่ ง   เ ป็ น   ก ลุ่ ม   ย่ อ ย   ข อ ง   ต ร ะ กู ล   ภ า ษ า   ข ร้ า  -   ไ ท  [ ]   สั น นิ ษ ฐ า น   ว่ า  [ ]   ภ า ษ า   ใ น   ต ร ะ กู ล   นี้   มี   ถิ่ น   ก ำ เ นิ ด   จ า ก   ท า ง   ต อ น   ใ ต้   ข อ ง   ป ร ะ เ ท ศ   จี น  [ ]   แ ล ะ   นั ก   ภ า ษ า ศ า ส ต ร์   บ า ง   ส่ ว น   เ ส น อ   ว่ า  [ ]   ภ า ษ า   ไ ท ย   น่ า   จ ะ   มี   ค ว า ม   เ ชื่ อ ม   โ ย ง   กั บ   ต ร ะ กู ล   ภ า ษ า   อ อ   ส โ ต ร  -   เ อ เ ชี ย   ติ ก  [ ]   ต ร ะ กู ล   ภ า ษ า   อ อ   ส โ ต ร นี เ ซี ย น  [ ]   แ ล ะ   ต ร ะ กู ล   ภ า ษ า   จี น  -   ทิ เ บ ต  [0x0a] 
Breaks:     c  c c c lc  c c c    lc  c  c c    lc  c c c lc  c c lc  c c c c    lc  c  c lc  c c c lc  c c c c c lc  c c lc  c c c lc  c c c c lc  c c  lc  c c lc  c c c c c lc  c c c    lc  c c c lc  c c lc  c  c lc  c c c lc  c lc  c   c lc  c c c lc  c lc    c lc  c  c lc  c   c lc   c c lc  c c lc  c c c  c lc  c c c lc  c  c c  lc  c c    lc   c c  c c c c lc   c c    lc  c c c lc  c lc  c c c  c lc    lc   lc    c lc  c c c  c lc  c c lc  c c lc  c c lc  c  lc  c c lc  c c c c c lc   c c    lc  c c lc   c lc  c c c c c c c c  lc  c c lc   c c lc  c c c lc   c c    lc  c c c lc  c c lc   c lc  c lc   lc  c c c lc  c   c c lc  c c lc   c lc  c c c  c lc  c c c lc  c lc  c c c c  lc  c c c  c lc   c c    lc  c c c  c lc  c c c lc  c lc  c c c c  c c  c c c    lc  c c lc  c c c  c lc  c c c lc   c c  lc   c c c c       c
Whitespace:                  x             x                                x                                                                                                          x                                                                                                                                                              x                             x                                                                                                                             x                                                                          x                                                                                                                                           x                                                       x                                                        w       w
Words:      bs b b b bse b b be   bse b  b be   bse b b b bse b b bse b b b be   bse b  b bse b b b bse b b b b b bse b b bse b b b bse b b b b bse b b  bse b b bse b b b b b bse b b be   bse b b b bse b b bse b  b bse b b b bse b bse b   b bse b b b bse b bse   b bse b  b bse b   b bse  b b bse b b bse b b b  b bse b b b bse b  b be bse b be   bse  b b  b b b b bse  b be   bse b b b bse b bse b b b  b bse   bse  bse   b bse b b b  b bse b b bse b b bse b b bse b  bse b b bse b b b b b bse  b be   bse b b bse  b bse b b b b b b b b  bse b b bse  b b bse b b b bse  b be   bse b b b bse b b bse  b bse b bse  bse b b b bse b   b b bse b b bse  b bse b b b  b bse b b b bse b bse b b b be bse b b b  b bse  b be   bse b b b  b bse b b b bse b bse b b b b  b b  b b be   bse b b bse b b b  b bse b b b bse  b be bse  b b b be      b
Sentences:  bs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            e       b
